### PR TITLE
Fixed typos and wording in step-20

### PR DESCRIPTION
**Pull request by nghayes**

### CODE REVIEW

File: examples/step-20/step-20.cc [hunk 1]
```diff
@@ -220,7 +220,7 @@ namespace Step20
   // real-life porous media flow simulations, and we would like to use the
   // opportunity to demonstrate the technique to use tensor valued functions.
   //
-  // Possibly unsurprising, deal.II also has a base class not only for scalar
+  // Possibly unsurprisingly, deal.II also has a base class not only for scalar
```
Comment (bangerth) [inline examples/step-20/step-20.cc]: ha, nice find!

**Comment by bangerth** [comment]
@nghayes -- let me know once you've added the changelog entry!

**Comment by nghayes** [comment]
@bangerth I think I have added it now! Let me know if there is anything I should change.

**Comment by drwells** [comment]
@nghayes Are you sure you pushed your updated branch to GitHub? I only see one changed file.

**Comment by nghayes** [comment]
@drwells I did not, but the second file should be there now. Thanks for catching that!

**Comment by bangerth** [comment]
OK to merge, but the required CI step is stuck at "pending".

**Pull request merged**